lib/web_ui/lib/src/engine/text/font_collection.dart

```diff
@@ -67,10 +67,13 @@ class HtmlFontCollection implements FontCollection {
             descriptors[descriptor] = '${fontAsset[descriptor]}';
           }
         }
-        _assetFontManager!.registerAsset(
+        _assetFontManager!.downloadAsset(
             family!, 'url(${assetManager.getAssetUrl(asset)})', descriptors);
       }
     }
+
+    final List<DomFontFace?> loadedFonts = await Future.wait(_assetFontManager!._fontLoadingFutures);
```

Comment:
just call `_assetFontManager!.downloadAllFonts()` here otherwise it won't work for the polyfill font manager